<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lLabel">
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="lDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="uLabel">
       <property name="text">
        <string>u</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="uDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="vLabel">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="vDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lBaseLabel">
       <property name="text">
        <string>lBase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="lBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="uBaseLabel">
       <property name="text">
        <string>uBase</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="uBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vBaseLabel">
       <property name="text">
        <string>vBase</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="vBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
